openmdao_book/features/core_features/working_with_components/distributed_components.ipynb

```diff
@@ -0,0 +1,482 @@
+{
```

Comment:
<p>we need to replace the <code>:ref:</code>  functionaility?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMDAO/OpenMDAO_Book/pull/78/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Once we're finished writing all the docs, that will be the final story to go through and make connections so we want to leave the `:ref:` tags in for now so we can go back and find them later